verify/verify-govet.sh

```diff
@@ -17,4 +17,6 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-find . -name "*.go" | grep -v "\/vendor\/" | xargs -n 1 go vet
+go get ./...
```

Comment:
(And I have to stop shaving yaks at some point, my whole goal is to fix races in mungegithub but I need go vet to be working to do that)